CONTRIBUTING.md

```diff
@@ -103,6 +88,15 @@ line of your file_.
 If you want to inherit a language, but don't want the languages inheriting from yours to inherit it,
 you can mark the language as optional (by putting it between parenthesis).
 
+#### Formating
+
+All queries are expected to follow a standard format, with every node on a single line and indented by two spaces for each level of nesting. You can automatically format the bundled queries by running the provided formatter `./scripts/format-queries.lua` from the repo root.
+
+Should you need to preserve a specific format for a node, you can exempt it (and all contained nodes) by placing before it
+```query
+; format-ignore
+```
```

Comment:
I don't know what needs to be fact-checked here :clinking_glasses:. Cannot be anymore correct than this